cmd/deploy.go

```diff
@@ -47,16 +46,21 @@ var deployCmd = &cobra.Command{ //nolint:gochecknoglobals
 		}
 
 		names := make([]string, len(integrations))
+		revisionIds := make([]string, len(integrations))
 		for idx, i := range integrations {
 			names[idx] = i.Name
+			revisionIds[idx] = i.LatestRevision.Id
 		}
 
 		if len(names) == 0 {
 			logger.Infof("No integrations were found in the source file.\n")
 		} else if len(names) == 1 {
-			logger.Infof("Successfully deployed your integration %s.\n", names[0])
+			logger.Infof("Successfully deployed your integration %s (revision ID: %s).\n", names[0], revisionIds[0])
```

Comment:
We might want to highlight in this message that this is something they should note down.